lxd/network_allocations.go

```diff
@@ -181,16 +182,26 @@ func networkAllocationsGet(d *Daemon, r *http.Request) response.Response {
 			}
 
 			for _, lease := range leases {
-				if shared.ValueInSlice(lease.Type, []string{"static", "dynamic"}) {
+				if shared.ValueInSlice(lease.Type, []string{"static", "dynamic", "uplink"}) {
 					cidrAddr, nat, err := ipToCIDR(lease.Address, netConf)
 					if err != nil {
 						return response.SmartError(err)
 					}
 
+					var allocationType, usedBy string
+					if lease.Type == "uplink" {
+						allocationType = "uplink"
+						networkName := strings.TrimSuffix(strings.Split(lease.Hostname, "/")[1], ".uplink")
```

Comment:
We should always test for the length coming back from strings.Split before assuming we can use element 1

strings.Cut would be more appropriate here i think

Reply:
Indeed `Cut` would be more appropriate, thanks for the insight.
Closing this because we are changing the approach to get the network name and project altogether.